src/ServiceControl.AcceptanceTests/ConfigureEndpointAzureServiceBusStandardTopologyTransport.g.cs

```diff
@@ -0,0 +1,36 @@
+﻿extern alias TransportASBS;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using ServiceBus.Management.AcceptanceTests;
+using ServiceControl.Transports.ASBS;
+
+public class ConfigureEndpointAzureServiceBusStandardTransport : ITransportIntegration
```

Comment:
Should this be named something like `ConfigureEndpointAzureServiceBusNetStandardTransport` instead, or is there something requiring this specific name?

Reply:
Can be any name. What ever makes us happy and singing in unison 🎶😊

Reply:
I picked it to coincide with the ASBS acronym we've chosen